### ISSUE TITLE
Retest all scripts & get running on Python 3

### ISSUE THREAD

**Issue by pgleeson** [openworm/Blender2NeuroML#4]

**Comment by TC-Wheel** [comment]
The tests are getting stuck on line 14 of the WrlToNeuroML.py - 
No module "neuroml" found. 
I think a dependency is missing in the test environment- and I think it would be harder (for me) to get the github test environment to actually function than to just follow the readme and do a local install. 
Since it's a legacy dev environment, and I've had bad experiences installing both old and new software on my computer- (I'm afraid the old version of blender will mess up my newer one) I'll probably just run it in a virtual machine. 
(I just wish I had access to VMware or something like I used in school. I'm sure I'll find something.)

**Comment by TC-Wheel** [comment]
I'm setting up an Ubuntu VirtualBox install. If I can't get that to work, then I'll try a Window 10 VM

**Comment by pgleeson** [comment]
@TC-Wheel The dependency that you mention that's missing can be installed with `pip install libneuroml`. 

You could also try installing miniconda for managing multiple versions of python: https://docs.conda.io/en/latest/miniconda.html

**Comment by TC-Wheel** [comment]
I ran the Python 2to3 script on the files, and am now debugging and seeing which ones still are working.
I will list them here and the errors that I'm currently getting. 

CheckWrlFile.py - Updated and working on Python 3.8.3 with no errors.
WrlToNeuroML.py - Currently still working on an error in line 134 on Entity.py "Lambda() 1 Missing Positional Argument: 'k'

**Comment by TC-Wheel** [comment]
To get WrlToNeuroML.py working:
I had to edit Entity.py and just found that the sorted dict output and unsorted dict output were the same. 
Maybe it's dumb, but just removing the sort on the for loop worked. 
It might be bad because the sort from a previous run could have changed the hashmap (dict) so that I'm benefitting from the sorting of a previous run.
To test this I would just have to do a clean download of the program. 
But isn't that exactly what the github tests are for? Maybe their output is messed up though- and they only test if the script runs. 

Change: 
Before: "for k1, value in sorted(brunches_temp.iteritems(),key=lambda (k,v):(len(v),k),reverse=True):"
Now: "for k1, value in brunches_temp.items():"

I also changed "iteritems()" to just item(). 

Maybe this is too much info in the wrong place, I'm just like a cave man inventing my own github culture.
